smarts/core/trap_manager.py

```diff
@@ -342,15 +333,16 @@ def _mission2trap(self, road_network, mission, default_zone_dist=6):
                 n_lanes=1,
             )
 
-        out_config = TrapConfig(
```

Comment:
Remove `TrapConfig`? Even the whole `trap_config.py` ?

Reply:
Certainly.